beta-src/src/utils/map/WDCountryHighlightFilters.tsx

```diff
@@ -2,63 +2,119 @@ import * as React from "react";
 import Country from "../../enums/Country";
 import webDiplomacyTheme from "../../webDiplomacyTheme";
 
+interface HoldConfig {
+  slope: number;
+  standardDeviation: number;
+}
+
 interface CountryHighlightFilter {
   country: Country;
   floodOpacity?: number;
+  hold: HoldConfig;
 }
 
 const countryHighlightFilterConfig: CountryHighlightFilter[] = [
   {
     country: Country.AUSTRIA,
+    hold: {
+      slope: 7,
+      standardDeviation: 2,
+    },
   },
   {
     country: Country.ENGLAND,
+    hold: {
+      slope: 7,
+      standardDeviation: 2,
+    },
   },
   {
     country: Country.FRANCE,
+    hold: {
+      slope: 7,
+      standardDeviation: 2,
+    },
   },
   {
     country: Country.GERMANY,
+    hold: {
+      slope: 7,
+      standardDeviation: 2,
+    },
   },
   {
     country: Country.ITALY,
+    hold: {
+      slope: 7,
+      standardDeviation: 2,
+    },
   },
   {
     country: Country.RUSSIA,
-    floodOpacity: 0.85,
+    hold: {
+      slope: 8,
+      standardDeviation: 1,
+    },
   },
   {
     country: Country.TURKEY,
+    hold: {
+      slope: 7,
+      standardDeviation: 2,
+    },
   },
 ];
 
 const WDCountryHighlightFilterDefs = function (): React.ReactElement[] {
   return countryHighlightFilterConfig.map((filter: CountryHighlightFilter) => (
-    <filter id={filter.country} key={filter.country}>
-      <feOffset dx="5" dy="5" in="SourceGraphic" result="shadow1" />
-      <feOffset dx="-5" dy="-5" in="SourceGraphic" result="shadow2" />
-      <feMerge result="offset-blur">
-        <feMergeNode in="shadow1" />
-        <feMergeNode in="shadow2" />
-        <feMergeNode in="SourceGraphic" />
-      </feMerge>
-      <feGaussianBlur stdDeviation="7.5" result="offset-blur" />
-      <feComposite
-        operator="out"
-        in="SourceGraphic"
-        in2="offset-blur"
-        result="inverse"
-      />
-      <feFlood
-        floodColor={webDiplomacyTheme.palette[`${filter.country}`].main}
-        floodOpacity={filter.floodOpacity || 1}
-        result="color"
-      />
-      <feComposite operator="in" in="color" in2="inverse" result="shadow" />
-      <feComponentTransfer in="shadow" result="shadow">
-        <feFuncA type="linear" slope="2.5" />
-      </feComponentTransfer>
-    </filter>
+    <>
```

Comment:
shouldn't need this `<>` since you already have a single tag `filter` below

Reply:
> shouldn't need this `<>` since you already have a single tag `filter` below

Not sure I'm following but there are two different filters being added per country so fragment is required.

Reply:
Didn't catch that thought it was only 1